packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -810,6 +809,37 @@ describe("Data Store Context Tests", () => {
         });
 
         describe("Initialization", () => {
+            const myObj: FluidObject = { fakeProp: "fakeValue" };
+            const registry: ISharedObjectRegistry = {
+                get(name: string) {
+                    throw new Error("Not implemented");
+                },
+            };
+            const dataStoreRuntime = new FluidDataStoreRuntime(
+                localDataStoreContext,
+                registry,
+                /* existing */ false,
+                (async (dataStoreRuntime) => requestFluidObject(dataStoreRuntime, "/")));
+
+
+            // Datastore should allow itself to attach, then after that throw on get, and then ensure it can summarize.
+            it("entryPoint is not initialized until calling request", async () => {
+                factory = {
+                    type: "store-type",
+                    get IFluidDataStoreFactory() { return factory; },
+                    instantiateDataStore: async (context: IFluidDataStoreContext, existing: boolean) =>
+                        dataStoreRuntime,
+                };
+                const dataStore = await factory.instantiateDataStore(localDataStoreContext, false);
+                await localDataStoreContext.attachRuntime(factory, dataStore);
+                assert((await dataStoreRuntime.entryPoint?.get()) === myObj, "entryPoint was not initialized after attaching.");
```

Comment:
myObjs is some random object that is initialized above and never passed around anywehere. I'm not sure how something can be equal to it.
Nit - I'd use same kind of asserting here and on line 840, i.e. either assert() or assert.strictEqual()